.changeset/little-cat-eating.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+Added table expandable animation.
```

Comment:
changeset not needed because the animation already existed in v2, so there is no "change" from a user perspective.